lib/ui/pages/account-linking/available_fsp.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/available_fsp_controller.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+
+class AvailableFSPScreen extends StatelessWidget {
+  Widget _buildListItem(String fspName) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(15, 10, 15, 10),
+      child: ShadowBox(
+        color: LightColor.navyBlue1,
+        child: ListTile(
+          trailing: const Icon(Icons.arrow_forward_ios),
+          title: Text(fspName),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildList() {
+    return Obx(() {
+      final AvailableFSPController fspController = Get.find<AvailableFSPController>();
+      if(fspController.fsps.value.isEmpty) {
+        return _buildEmptyDisplay();
+      }
+
+      return ListView.builder(
+        itemCount: fspController.fsps.value.length + 2,
+        itemBuilder: (BuildContext ctxt, int index) {
+          switch(index) {
+            case 0:  return _buildIcon(); break;
```

Comment:
Does the linter allow multiple syntax on a single line?

Reply:
Probably not, not sure, I'll just format the whole file